docs/README.md

```diff
@@ -0,0 +1,75 @@
+- Backend & utility scripts
+
+## API Directory Structure
+```
+.
+├── api
+│   └── auth                Authentication code for API
+│   └── db
+│       └── models          DB model definitions            
+│       └── migrations      DB migration configs
+│           └── versions    The DB migrations
+│   └── logging
+│   └── route               API route definitions
+│       └── handler         API route implementations
+│   └── scripts             Backend scripts that run separate from the application
+│   └── util                Utility methods and classes useful to most areas of the code
+│
+├── tests
+├── Dockerfile          Multi-stage Docker build file for project
+├── docker-compose.yml  Config file for docker-compose tool, used for local development
+├── local.env           Environment variable configuration for local files
+├── openapi.yml         API specification
+├── pyproject.toml      Python project configuration file
+└── setup.cfg           Python config for tools that don't support pyproject.toml yet
+```
+
+# Getting started
+
+1. In your terminal, `cd` to this repo.
+2. Make sure you have [Docker Desktop](https://www.docker.com/products/docker-desktop/) installed & running.
+3. Run `make init start` to build the image and start the container.
+4. Navigate to `localhost:8080/v1/docs` to access the Swagger UI.
+5. Run `make run-logs` to see the logs of the running API container
+6. Run `docker-compose down` when you are done to delete the container.
+
+# Information
+
+* [API Technical Overview](/docs/app/technical-overview.md)
+* [Database Migrations](/docs/app/database-migrations.md)
+* [Formatting and Linting](/docs/app/formatting-and-linting.md)
+* [Writing Tests](/docs/app/writing-tests.md)
+
+## Additional Useful Commands
+`make test` will run all of the tests. Additional arguments can be passed to this command which will be passed to pytest like so: `make test args="tests/api/route -v"` which would run all tests in the route folder with verbosity increased. See the [Pytest Docs](https://docs.pytest.org/en/7.1.x/reference/reference.html#command-line-flags) for more details on CLI flags you can set.
+
+`make clean-volumes` will spin down the docker containers + delete the volumes. This can be useful to reset your DB, or fix any bad states your local environment may have gotten into.
+
+See the [Makefile](./app/Makefile) for a full list of commands you can run.
+
+## Docker and Native Development
+Several components like tests, linting, and scripts can be run either inside of the Docker container, or outside on your native machine.
+Running in Docker is the default, but on some machines like the M1 Mac, running natively may be desirable for performance reasons.
+
+You can switch which way many of these components are run by setting the `PY_RUN_APPROACH` env variable in your terminal.
+* `export PY_RUN_APPROACH=local` will run these components natively
+* `export PY_RUN_APPROACH=docker` will run these within Docker
```

Comment:
I prefer running in docker as the default, but that's not actually what we have in the Makefile. Can we update the Makefile to make docker the default?

Reply:
I thought I switched that at some point based on a prior PR, will fix.